plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -39,14 +38,13 @@ func TestWinPerfcountersConfigGet1(t *testing.T) {
 
 	perfobjects[0] = PerfObject
 
-	m := Win_PerfCounters{PrintValid: false, TestName: "ConfigGet1", Object: perfobjects}
```

Comment:
TestName was also removed in this pull request https://github.com/influxdata/telegraf/pull/2800